flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/MultipleInputStreamTask.java

```diff
@@ -251,12 +251,8 @@ protected void createInputProcessor(
                 },
                 "stop chained Flip-27 source for stop-with-savepoint --drain");
 
-        return assertTriggeringCheckpointExceptions(
-                sourcesStopped.thenCompose(
-                        ignore ->
-                                triggerSourcesCheckpointAsync(
-                                        checkpointMetaData, checkpointOptions)),
-                checkpointMetaData.getCheckpointId());
+        return sourcesStopped.thenCompose(
```

Comment:
Hi Dawid I might not fully understand the change since it seems FLINK-23471 does not change the behavior of triggering checkpoint, and only modifies the behavior of `notifyCheckpointComplete / notifyCheckpointAbort` ?

Reply:
Sorry, I made a mistake in the issue number. Will fix that shortly. I meant FLINK-23741.